llvm/tools/sycl-post-link/SYCLDeviceLibReqMask.cpp

```diff
@@ -190,11 +190,13 @@ uint32_t getDeviceLibBits(const std::string &FuncName) {
                             DeviceLibExt::cl_intel_devicelib_assert)));
 }
 
+} // namespace
+
 // For each device image module, we go through all functions which meets
 // 1. The function name has prefix "__devicelib_"
 // 2. The function is declaration which means it doesn't have function body
 // And we don't expect non-spirv functions with "__devicelib_" prefix.
-uint32_t getModuleDeviceLibReqMask(const Module &M) {
+uint32_t llvm::getSYCLDeviceLibReqMask(const Module &M) {
```

Comment:
I'm not sure the `llvm::` prefix is still required here and for the `Triple` class (the same as it isn't used for the `Module` class here). There is the `using namespace llvm;` directive at the top of the file.

Reply:
`llvm::` required for the function definition, otherwise it will not link. It's not needed for `Triple` class, I agree.

Reply:
Ok. I decided, that for single function better to use just namespace prefix.

I checked and have found this rule: https://llvm.org/docs/CodingStandards.html#use-namespace-qualifiers-to-implement-previously-declared-functions